package.json

```diff
@@ -72,9 +73,22 @@
     "svelte2tsx": "^0.6.0",
     "tailwindcss": "^3.2.4",
     "ts-jest": "^29.0.3",
+    "ts-patch": "^2.1.0",
     "tslib": "^2.4.1",
     "typescript": "^4.9.4",
+    "typescript-transform-extensions": "^1.0.1",
     "vite": "^4.0.3",
     "vitest": "^0.25.8"
-  }
+  },
+  "exports": {
```

Comment:
is this change created by `npx svelte-migrate package`?

Reply:
I did the migration by hand, but yes I understand that package 2.0.0 and up use exports like this (instead of creating the previous `/package` directory we saw previously)

Reply:
actually I think there ought to be an import in there as well